CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 This file is used to list changes made in each version of the homebrew cookbook.
 
-## Unreleased
+## 5.4.6 - *2024-05-06*
```

Comment:
don't include the version bump, sous-chefs automation takes care of that. Just need the ##Unreleased header and description of changes under it.